Automattic/jetpack
# CLI: Add release-branch to jetpack release

#### Changes proposed in this Pull Request:

This PR adds `release-branch` to the `jetpack release` command so you can create a release branch using the CLI tool.

#### Jetpack product discussion



#21160

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






Kinda tricky to test, since the script has a lot of error handling, but ideally:

* Run `jetpack release` and follow the prompts to create a release branch for a project.
* It should automatically suggest a potential version, but you can enter your own.
* Try passthroughs - `jetpack release plugins/jetpack -b release-branch` should run the release branch script for 10.4-beta, etc.
* Don't push any branches you create.

Might be easier to merge and put to the test during the 10.4 release later this month. Although I notice the 10.4 branch was already created.
